assets/scss/_variables_project.scss

```diff
@@ -49,6 +53,8 @@ $navbar-dark-disabled-color: rgba($black, 0.25);
     font-size: 1.5em;
     font-weight: 500;
     line-height: 1.2;
+    margin-left: 0;
+    margin-right: 0;
```

Comment:
Not really new in this PR, but if this is scss format you should be able to nest these `.banner` child blocks inside the main `.banner` block, like:

```
.banner {
  margin: 3em 1em;
  text-align: center;

  > .banner-image {
    margin-bottom: 2em; 
    margin-left: auto;
    margin-right: auto;
    max-width: 65%;
  }

  > .banner-text {
    /* and so on */
} /* then close the main .banner block */
```

Reply:
I've updated the file, so PTAL.